docs/examples/configurations.yaml

```diff
@@ -0,0 +1,53 @@
+    "spark.executor.extraJavaOptions": "-verbose:gc -XX:+PrintGCDetails -XX:+PrintGCDateStamps -XX:+UseConcMarkSweepGC -XX:CMSInitiatingOccupancyFraction=70 -XX:MaxHeapFreeRatio=70 -XX:+CMSClassUnloadingEnabled -XX:OnOutOfMemoryError='kill -9 %p' -Dhttp.proxyHost='$PROXY_ADDRESS' -Dhttp.proxyPort='3128' -Dhttp.nonProxyHosts='$NO_PROXY_ADDRESSES' -Dhttps.proxyHost='$PROXY_ADDRESS' -Dhttps.proxyPort='3128'"
+    "spark.driver.extraJavaOptions": "-XX:+UseConcMarkSweepGC -XX:CMSInitiatingOccupancyFraction=70 -XX:MaxHeapFreeRatio=70 -XX:+CMSClassUnloadingEnabled -XX:OnOutOfMemoryError='kill -9 %p' -Dhttp.proxyHost='$PROXY_ADDRESS' -Dhttp.proxyPort='3128' -Dhttp.nonProxyHosts='$NO_PROXY_ADDRESSES' -Dhttps.proxyHost='$PROXY_ADDRESS' -Dhttps.proxyPort='3128'"
+    "spark.sql.warehouse.dir": "s3://$S3_PUBLISH_BUCKET/analytical-dataset/hive/external"
+- Classification: "spark-hive-site"
+  Properties:
+    "hive.metastore.client.factory.class": "com.amazonaws.glue.catalog.metastore.AWSGlueDataCatalogHiveClientFactory"
+- Classification: "hive-site"
+  Properties:
+    "hive.metastore.schema.verification": "false"
+    "hive.metastore.client.factory.class": "com.amazonaws.glue.catalog.metastore.AWSGlueDataCatalogHiveClientFactory"
+    "hive.metastore.warehouse.dir": "s3://$S3_PUBLISH_BUCKET/analytical-dataset/hive/external"
+- Classification: "hbase-site"
+  Properties:
+    "hbase.server.keyvalue.maxsize": "750000000"
+    "hbase.bulkload.retries.retryOnIOException": "true"
+    "hbase.client.write.buffer": "8388608"
+    "hbase.regionserver.handler.count": "4"
+    "hbase.client.keyvalue.maxsize": "750000000"
+    "hbase.bucketcache.bucket.sizes": "5120,9216,17408,33792,41984,50176,58368,66560,99328,132096,197632,263168,394240,525312,656384,787456,918528,1049600,1180672,1311744,1442816,1704960,1967104,2229248"
+    "hbase.rootdir": "s3://$S3_INGEST_BUCKET/business-data/single-topic-per-table-hbase"
+- Classification: "hbase"
+  Properties:
+    "hbase.emr.storageMode": "s3"
+    "hbase.emr.readreplica.enabled": "true"
+- Classification: "emrfs-site"
+  Properties:
+    "fs.s3.consistent.retryPeriodSeconds": "3"
+    "fs.s3.consistent": "false"
+    "fs.s3.cse.materialsDescription.enabled": "true"
+    "fs.s3.consistent.retryCount": "5"
+    "fs.s3.consistent.metadata.tableName": "DataGen-metadata"
+- Classification: "spark-env"
+  Configurations:
```

Comment:
I think this is a simple copy-paste mistake? Don't need the additional `Configurations` level here, so the `Classification` underneath can be outdented to the same level as the rest.

Reply:
OK, Gordon explained this and it's a spark-env classification oddity.